_posts/2020-05-22-status-update.md

```diff
@@ -0,0 +1,79 @@
+
+Thank you to [https://github.com/vchepkov](Vadym Chepkov) for your [Docker PR](https://github.com/puppetlabs/puppetlabs-docker/pull/613). Housekeeping on our modules is always beneficial and we really appreciate you taking the time to contribute.
+
+## PDKSync Runs
+
+ [David](https://github.com/david22swan) has ran 2 PDKSync runs this week to include generic template updates in our modules and to add Ubuntu 20.04 support and [Danny](https://github.com/carabasdaniel) has been running one to switch our default provisioner from vmpooler to abs.
+
+## Making it Possible to Visualize Data
+
+[David](https://github.com/DavidS) has been working hard on making use of [honeycomb](https://ui.honeycomb.io/)to allow us to visualize data on pretty dashboards. Our new dashboard this week will track our migration from vmpooler to abs. Super exciting that we can now see dashboards and drill down rather than scrolling through large log files.
+
+## The IAC Team Adapting to the New 'Norm'
+
+Everyone in the team seems to have started adapting to the new ‘norm’ of working from home every day. It seems everyone has started trying out new things. Here are some of the things we have been getting up to:
+
+[David](https://github.com/DavidS) has been taking Oscar and Miku out walking almost daily, soaking up the lovely views around Belfast and of course playing [factorio](https://factorio.com/) for way too long. Can we blame him when confined to the house, lets just refer to it mastering skills online.
+
+[Norman](https://github.com/DavidS) over the past year has has a lot of landscaping carried out in his garden therefore he finds himself spending more time in his garden than before. With the weather we have been having in Belfast recently he would be crazy to miss the opportunity.
+
+[Ciaran](https://github.com/sanfrancrisko) now knows his neighborhood by the back of his hand, has taken up more cooking opportunities at home (he currently has enough Bolognese in his freezer to feed the whole IAC team), is getting a big nostalgia hit playing Streets of Rage 4 online with his mate and has big plans on getting back into playing guitar, it just hasn’t happened yet. Sounds like Ciaran has been doing great at keeping busy!
+
+[Danny](https://github.com/carabasdaniel) has found himself spending more time chilling out and reading books and when he is not doing that he is ‘fine tuning’ his video gaming skills.
+
+[David](https://github.com/david22swan) carries out a vigorous home workout regime on Tuesday, Wednesday and Thursdays. Way to go David, serious commitment there!
+
+[Sheena](https://github.com/sheenaajay) is currently home schooling 2 lovely girls. Although when they have some down time she is exploring new recipes with them.
+
+[Diana](https://github.com/daianamezdrea) has started to do some home workouts and has started eating healthy I wonder is this a result of mastering how to bake a perfect lava cake? I also wonder how shipping a lava cake would work!?
```

Comment:
```suggestion
[Diana](https://github.com/daianamezdrea) has started to do some home workouts and has started eating healthy. I wonder if this is a result of mastering how to bake a perfect lava cake? I also wonder how shipping a lava cake would work!?
```

Reply:
...and one more `s/Diana/Daiana/` there too 😉